ansible/roles/fanout/tasks/fanout_mlnx.yml

```diff
@@ -6,8 +6,22 @@
 ### specified in this playbook, you would need to come up with your own fanout switch deployment
 ### playbook
 ################################################################################################
+- name: set login to tacacs if tacacs is defined
+  set_fact: ansible_ssh_user={{ fanout_tacacs_mlnx_user }} ansible_ssh_password={{ fanout_tacacs_mlnx_password }}
+  when: >
+    fanout_tacacs_user is not defined and fanout_tacacs_user is not defined and
```

Comment:
Why not check fanout_tacacs_mlnx_* only?

Reply:
currently tacacs is enabled for sonic fanout only, I want to leave some room for future configs.

Reply:
@Blueve